Sources/docc/DocCDocumentation.docc/DocC.symbols.json

```diff
@@ -3478,7 +3478,7 @@
             "text" : "For example, use the page image directive to customize the icon used to represent this page in the navigation sidebar,"
           },
           {
-            "text" : "or the card image used to represent this page when using the ``Links`` directive and the ``Links\/detailedGrid``"
+            "text" : "or the card image used to represent this page when using the ``Links`` directive and the ``Links\/VisualStyle\/detailedGrid``"
```

Comment:
These documentation changes was done before this PR but we hadn't updated the DocC SGF to reflect the latest documentation changes.